api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirDiagnosticReportDaoImpl.java

```diff
@@ -11,19 +11,28 @@
 
 import static org.hibernate.criterion.Restrictions.eq;
 
+import java.util.Collection;
+
 import lombok.AccessLevel;
 import lombok.Setter;
+import org.hibernate.Criteria;
 import org.hibernate.SessionFactory;
 import org.openmrs.Obs;
 import org.openmrs.api.db.DAOException;
 import org.openmrs.module.fhir2.api.dao.FhirDiagnosticReportDao;
+import org.openmrs.module.fhir2.api.search.param.SearchParameterMap;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.stereotype.Component;
 
 @Component
 @Setter(AccessLevel.PACKAGE)
-public class FhirDiagnosticReportDaoImpl implements FhirDiagnosticReportDao {
+public class FhirDiagnosticReportDaoImpl extends BaseFhirDao<Obs> implements FhirDiagnosticReportDao {
+	
+	/**
+	 * 
+	 */
+	private static final long serialVersionUID = 1L;
```

Comment:
For right now, I'd actually prefer not to implement a serialVersionUID for Dao objects. They aren't stable enough.

Reply:
You are Wecome @ibacher  thanks alot.